src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -67,21 +67,22 @@
         >
           {{ item.index + 1 }} of {{ item.siblingsCount }} symbols inside
         </span>
-        <Reference
```

Comment:
Do you know if there was a specific reason we were using `Reference` instead of `router-link` before?

If so, maybe it would be more appropriate to create an `OptionalReference` that uses `Reference` instead of `router-link`